.buildkite/dagster-buildkite/dagster_buildkite/steps/integration.py

```diff
@@ -60,12 +64,12 @@ def build_backcompat_suite_steps() -> List[BuildkiteTopLevelStep]:
     )
 
 
-def backcompat_extra_cmds(_, factor: str) -> List[str]:
+def backcompat_extra_cmds(_, factor: Optional[str]) -> List[str]:
     tox_factor_map = {
         "user-code-latest-release": LATEST_DAGSTER_RELEASE,
         "user-code-earliest-release": EARLIEST_TESTED_RELEASE,
     }
-
+    assert factor
     webserver_version = DAGSTER_CURRENT_BRANCH
```

Comment:
?

Reply:
aligning the typing [1] makes `factor` `Optional` but we must always be getting it in this context so i just asserted to make the type checker happy